docs/Concepts/Consensus-Protocols/Comparing-PoA.md

```diff
@@ -50,23 +49,23 @@ redundancy if the validator fails.
 
 ### Liveness
 
-Clique is more fault tolerant than IBFT 2.0 and QBFT. Clique tolerates up to half of the validators
-failing. IBFT 2.0 and QBFT networks require greater than or equal to two-thirds of validators to be
-operating to create blocks. For example, an IBFT 2.0 or QBFT network of:
+Clique is more fault tolerant than QBFT and IBFT 2.0. Clique tolerates up to half of the validators
+failing. QBFT and IBFT 2.0 networks require greater than or equal to two-thirds of validators to be
+operating to create blocks. For example, an QBFT and IBFT 2.0 network of:
 
-* Four to five validators tolerates one unresponsive validator
+* Four to five validators tolerates one unresponsive validator.
 * Six to eight validators tolerates two unresponsive validators.
 
 Networks with three or less validators can produce blocks but do not guarantee finality when
 operating in adversarial environments.
 
 !!! important
 
-    Using IBFT 2.0 or QBFT networks with three nodes for production purposes is not recommended.
+    We don't recommend using QBFT or IBFT 2.0 networks with three nodes for production purposes.
```

Comment:
Perhaps rephrase to say what we recommend? 

"We recommend using QBFT or IBFT 2.0 networks with at least four nodes in production environments."